app/views/mailing_list/steps/completed/_first_or_second_year_high_commitment.html.erb

```diff
@@ -0,0 +1,13 @@
+<p>
+  Learn more about <a href="/funding-and-support">funding and the financial support</a> available.
+</p>
+
+<p>
+  Get your questions answered and speak to current teachers at an <a href="/events">an event near you or online</a>.
+</p>
+
+<p>
+  Get help to discover whether teaching is right for you with an
```

Comment:
```suggestion
Discover whether teaching is right for you with the help of an
```
@philspray Worth rejigging this slightly so it is less repetitive? (you can probably reword better!)